resources/css/button.css

```diff
@@ -1,4 +1,4 @@
-.btn, button, [type="button"], [type="reset"], [type="submit"] {
```

Comment:
While popular in CSS frameworks like bootstrap and Daisy UI (where this is from) it's too greedy for our use

Reply:
We have a few of these global style overrides lurking, hopefully we can systematically remove them over time.